dataland-internal-storage/src/main/kotlin/db/migration/V2__MigrateEuTaxonomyNonFinancialsWithAbsoluteValues.kt

```diff
@@ -0,0 +1,77 @@
+package db.migration
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.dataland.datalandbackend.openApiClient.model.DataTypeEnum
+import org.flywaydb.core.api.migration.BaseJavaMigration
+import org.flywaydb.core.api.migration.Context
+import org.json.JSONObject
+
+/**
+ * This migration script updates the EU Taxonomy for non-financials data model switching
+ * from a field called value for percentages only to a structure holding the absolute value of a cash flow type as well
+ */
+@Suppress("ClassNaming")
```

Comment:
Any possibility to also change the linter settings to not make this Suppress statement necessary?

Reply:
This should be possible. I'll look into it.

Reply:
Das war ja einfach. Da war sogar schon eine RegEx angelegt.